build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
 
     dependencies {
         classpath pluginDependencies.gradle
+        classpath pluginDependencies.mapboxSDKVersions
```

Comment:
nit: `mapboxSDKVersions` -> `mapboxSdkVersions`, could be annoying to change at this point but something to consider in the future